input/pages/profiles.md

```diff
@@ -71,23 +71,23 @@ To support packaging, testing, and distribution of measure and library artifacts
 
```

Comment:
Please also add hyperlinks to the profiles listed under the section 7.2 Library Profile Usage.
CQFMComputableMeasure SHALL use CQLLibrary
CQFMExecutableMeasure SHALL use either ELM JSON Library and/or ELM XML Library
CQFMPublishableMeasure SHOULD use CRMIPublishableLibrary
CQFMShareableMeasure SHOULD use CRMIShareableLibrary